app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,2 @@
+class CommentsController < ApplicationController
+end
```

Comment:
- [ ] Kindly remove this comments controller file, as there is no matching route for it.

Reply:
> * [ ]  Kindly remove this comments controller file, as there is no matching route for it.

- [ ] Kindly remove this as requested by the previous reviewer.